application/src/Weekend.php

```diff
@@ -4,49 +4,77 @@
 
 class Weekend
 {
+    const MAPPING = [
+        '\o/'  => '🎉',
+        '◄:•D' => '🥳',
+        'B-)'  => '😎',
+        '¬‿¬'  => '😏',
+        ':('   => '😟',
+        'X-('  => '😠',
+    ];
+
+    /**
+     * Compute the main text with emojis
+     *
+     * @return string
+     */
+    public function getRichText()
+    {
+        $emoticons = array_keys(self::MAPPING);
+        $emojis = array_values(self::MAPPING);
+
+        return str_replace($emoticons, $emojis, $this->getText());
```

Comment:
```suggestion
        return strtr($emojis, $this->getText(), self::MAPPING);
```

Reply:
Nice! I didn't recall this function, but that's what I was looking for indeed. 👍